quiz.js

```diff
@@ -37,7 +51,27 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+  name: 'stark'
+}, {
+  $push: { people: {
+    peopleName: 'ned', 'arya', 'sansa'}
+  }
+});
+
+db.houses.update({
+  name: 'Targaryen'
+}, {
+  $push: { peopleName: 'Viserys', 'Daenerys'}
+  }
+});
 
```

Comment:
These first two are not correct. The objects we're creating are key-value pairs, and if we try to associate more than one value with a value, Mongo complains. 

Two viable ways to do this - either (1) add each person's name one at a time, or (2) use the $each operator, like so:
```
db.houses.update( { name: 'stark' }, { $push : { peopleName : { $each : 
  [
    'ned',
    'arya',
    'sansa'
  ] 
} } } );
db.houses.update( { name: 'Targaryen' }, { $push : { peopleName : { $each : 
  [
    'Viserys',
    'Daenerys'
  ] 
} } } );
```